pkg/models/capturetraffic.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+import "time"
+
+type TrafficCaptureParams struct {
+	Path             string
+	ProxyPort        uint32
```

Comment:
TrafficCaptureConfig would be better name

Reply:
Refactored